src/send_handler/main_send_handler.py

```
@@ -1,4 +1,4 @@
-from typing import Any, Dict, List
+from typing import List
```

Ruff (0.15.0)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)